tsdb/config.go

```diff
@@ -120,6 +123,11 @@ type Config struct {
 	// be compacted less frequently, store more series in-memory, and provide higher write throughput.
 	MaxIndexLogFileSize toml.Size `toml:"max-index-log-file-size"`
 
+	//  is the number items that can be cached within the TSI index. TSI caching can help
+	// with query performance when the same tag key/value predicates are commonly used on queries.
+	// Setting TSICacheSize disables the cache.
+	TSICacheSize int `toml:"tsi-cache-size"`
+
```

Comment:
Why is the field called `TSICacheSize`, the default called `DefaultSeriesIDSetCacheSize`, and the underlying field in `tsi1.Index` called `tagValueCacheSize`? Can we standardize them all to "series id set cache size" or "tag value cache size"? The `TSICacheSize` seems limiting since there could be other caches added.

Reply:
SGTM